Sources/App/Commands/Analyzer.swift

```diff
@@ -537,3 +538,24 @@ func updateLatestVersions(on database: Database,
                 .map { package }
         }
 }
+
+
+/// Event hook to run logic when new (tagged) versions have been discovered in an analysis pass. Note that the provided
+/// transaction could potentially be rolled back in case an error occurs before all versions are processed and saved.
+/// - Parameters:
+///   - client: `Client` object for http requests
+///   - transaction: database transaction
+///   - versions: array of newly discovered versions
+/// - Returns: future wrapping operation(s)
+func onNewVersions(client: Client,
+                   transaction: Database,
+                   versions: [Version]) -> EventLoopFuture<Void> {
+    let posts = versions
+        .filter { $0.reference?.isTag ?? false }
```

Comment:
NB: this is redundant, because 

```swift
let semVer = version.reference?.semVer
```

in `firehostPost` will return `nil` unless it's a tag but this avoids some extra lookups and makes that clearer.

However, we may actually _want_ to post default branch changes - it's a firehose after all (not that they're all that frequent though). We could do that with a hashtag `#latest` or something so people can mute it. Or even add those in general, `#beta` & `#stable`?